ml-agents/mlagents/trainers/ghost/controller.py

```diff
@@ -26,6 +26,8 @@ def __init__(self, maxlen: int = 10):
         self._learning_team: int = -1
         # Dict from team id to GhostTrainer for ELO calculation
         self._ghost_trainers: Dict[int, GhostTrainer] = {}
+        # Signals to the trainer control to perform a hard reset
+        self._reset = False
```

Comment:
nit: Either `_should_reset` or `_changed_teams` (I think I prefer the latter).

Reply:
or I guess `_changed_training_team` is closest to the method name.

Reply:
Done.